base/server/cms/src/com/netscape/cms/profile/common/EnrollProfile.java

```diff
@@ -210,7 +210,7 @@ public IProfileContext createContext() {
             // catch for invalid request
             cmc_msgs = parseCMC(locale, cert_request, donePOI);
             SessionContext sessionContext = SessionContext.getContext();
-            String authenticatedSubject = 
+            String authenticatedSubject =
```

Comment:
If we're not otherwise changing this file, perhaps leave the space? Keeps the diff pretty :)

Reply:
Took care of it.